starport/services/network/networkchain/prepare.go

```diff
@@ -66,7 +67,16 @@ func (c Chain) Prepare(ctx context.Context, gi networktypes.GenesisInformation)
 	}
 
 	// reset the saved state in case the chain has been started before
-	return cmd.UnsafeReset(ctx)
+	if err := cmd.UnsafeReset(ctx); err != nil {
+		return err
+	}
+
+	fmt.Printf(`
+🎉Chain is prepared for launch.
+
+You can start your node by running: %s start --home %s
```

Comment:
```suggestion
%s You can start your node by running: %s start --home %s
```

clispinner.Bullet

Reply:
It'd be hard to copy the command here because user needs to select it carefully. Instead, what if we print it to dedicated line? Then, user can just double click and copy.

Reply:
By double clicking it is possible to select only a word in output. Or maybe there are some special chars to change selection behaviour.

Reply:
Yes, double click selects a world and triple one selects the whole line which seems easier.